pybamm/experiments/experiment.py

```diff
@@ -66,7 +68,6 @@ def __init__(
         drive_cycles={},
     ):
 
-        import numpy as np
         self._np = np
```

Comment:
since we now import numpy at the top we should use that everywhere instead of declaring self._np

Reply:
Done.